src/content.ts

```diff
@@ -0,0 +1,56 @@
+import { Bee } from '@ethersphere/bee-js'
+import client from 'prom-client'
+import type { ContentConfig } from './config'
+import { logger } from './logger'
+import { register } from './metrics'
+
+const contentReuploadCounter = new client.Counter({
+  name: 'content_reupload_counter',
+  help: 'How many pinned content items were uploaded',
+})
+register.registerMetric(contentReuploadCounter)
+
+export class ContentManager {
+  private interval?: ReturnType<typeof setInterval>
+  private isReuploading?: boolean = false
```

Comment:
```suggestion
  private isReuploading: boolean = false
```

Since you are setting the default value, I don't think this needs to be optional.